### PR TITLE
Update blogs.yml

### PR DESCRIPTION
**Pull request by oorjadandotiya**
- url: https://technoorja.wordpress.com/2018/01/01/technoorja-learns-to-code/
#   title: Technoorja Learns To Code
#   img: Oorja_modified.jpeg
#   author_name: Oorja Dandotiya

Please read and understand everything below
**Do not delete any text other than where you are instructed.**

**Students: If one of them is applicable to you. Please check it.**

Check by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Read and understood (see CONTRIBUTING.md)
- [ ] Included a Preview link and screenshot showning after and before the changes.
- [x] Images are `240 x 240` [w x h].
- [ ] Included a description of change below.
- [ ] Squashed the commits.

# Changes done in this Pull Request

- If your change will be reflected on the website, please provide a **Test-Link** (**Hint : `gh-pages`**)
- Fixes`#<Issue no.>`

### CODE REVIEW

**Review by isuruAb — COMMENTED** [review]
where did you add this "Oorja_modified.jpeg" image

**Review by nikit19 — CHANGES_REQUESTED** [review]
Please write it somewhere else without # 
Please go through the readme.md and see how to add a blog

File: _data/blogs.yml [hunk 1]
```diff
@@ -2,10 +2,10 @@
 # This is to remove conflicts with other people adding entries.
 
 # The Format is :
-# - url: URL to the blog
-#   title: Title of the blog
-#   img: Image of the blog
-#   author_name: Name of the author of the blog (most likely you)
+# - url: https://technoorja.wordpress.com/2018/01/01/technoorja-learns-to-code/
```
Comment (0x48piraj) [inline _data/blogs.yml:5]: Please remove the comments

Comment (0x48piraj) [reply _data/blogs.yml:5]: I meant remove `#`

**Review by 0x48piraj — CHANGES_REQUESTED** [review]
See the code review

**Review by anshumanv — COMMENTED** [review]
Please don't remove the comments, add an image.

**Review by akshatnitd — CHANGES_REQUESTED** [review]
Please don't remove the comments, those are there to show the correct format in which the blogs are to be added. Do check the way others have added it, and add an entry of your blog similarly. Also, you need to upload the image in the `img/blog/` folder.

**Comment by ParthS007** [comment]
closing this as a duplicate of #785

**Pull request closed without merging**